scripts/update_dashboard_state.py

```diff
@@ -0,0 +1,21 @@
+import argparse
+import json
+from pathlib import Path
+from urllib.parse import urlparse
+
+import requests
+
+parser = argparse.ArgumentParser()
+parser.add_argument("url")
```

Comment:
we could default this to `"http://localhost:9999/"`, since that's usually where it'll be